### ISSUE TITLE
A11y_ML .NET Model Builder_Image Classification-Next Steps_Keyboard: 'Train in Azure' option is not accessible using keyboard. Keyboard focus is not landing on 'Train in Azure' option while navigating using tab key.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1630]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG2.1.1; #Keyboard; #A11yPreview; #A11yAuto; #TabStops;

### Environment Details:
Microsoft Visual Studio

### Re-pro Steps:

1. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Image classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In Data tab, Tab till "Browse" button and select the input file
7. After uploading data, click next step button, start training in Train tab
8. After completing training, click next step button, complete all the steps and navigate to 'Next Steps' tab
9. Verify whether 'Train in Azure' option is not accessible using keyboard or not

### Actual Result:
'Train in Azure' option is not accessible using keyboard. Keyboard focus is not landing on 'Train in Azure' option while navigating using tab key.

### Expected Result:
'Train in Azure' option should be accessible using keyboard. Keyboard focus should land on 'Train in Azure' option while navigating using tab key.

### User Impact:
Keyboard users will not get to access train in azure option if keyboard focus does not land on it.

**Comment by NiharikaTipirishetty** [comment]
@zewditu Issue is partially fixed. Train in Azure option is receiving keyboard focus but on pressing enter/space key, no action is getting performed. When train in azure optioned is selected using mouse, it will be navigated to training environment tab.


https://user-images.githubusercontent.com/83570638/131464815-a5435819-9860-4f26-9b58-09d497dc68a9.mp4

**Comment by beccamc** [comment]
@zewditu Please take a look at this issue.

**Comment by zewditu** [comment]
@NiharikaTipirishetty  I am not able to reproduce the issue in our latest main or model builder version ![image.png](https://images.zenhubusercontent.com/5d014b8c2b5aa11c6166d13d/aaf7c098-495b-4514-8b73-c799571a4a05).
Could you update your model builder ? 
But I am not sure why you got partially fixed.

**Comment by beccamc** [comment]
New build is available here https://devdiv.visualstudio.com/_apis/resources/Containers/8663755/VSSetup_Release17?itemPath=VSSetup_Release17%2FMicrosoft.ML.ModelBuilder.Gallery.Dev17.vsix

**Comment by NiharikaTipirishetty** [comment]
@beccamc @zewditu the new model builder version is throwing error. please suggest what need to be done.
I am using Visual Studio Version 17.0.0 Preview 3.1
![1630_error (2)](https://user-images.githubusercontent.com/83570638/133092036-8dad33da-de67-45a7-af78-ccaa8fa91f67.png)
![1630_error](https://user-images.githubusercontent.com/83570638/133092039-0e3bda66-b589-4988-ae2e-9035b143262a.png)

**Comment by beccamc** [comment]
@NiharikaTipirishetty sorry about this problem. 

Can you install Preview 5 (internal build) from this link?  https://aka.ms/vs/17/pre/vs_Enterprise.exe.  Using this version with the model builder extension above worked for me. 

![image](https://user-images.githubusercontent.com/9122518/133181528-b5a70b7a-5fa4-435c-ac3a-0a858293a3f0.png)

You should be able to validate all of our "Ready for review" issues with this build.

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed. Train in Azure option is receiving keyboard focus and on pressing enter key, it will be navigated to training environment tab.  Train in Azure option is keyboard accessible. Hence closing this bug.


https://user-images.githubusercontent.com/83570638/133382952-23d63c5f-e479-47c1-8932-b6d752455306.mp4
